src/Beman/Example/tests/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+include(GoogleTest)
+
+set(TARGET_LIBRARY_TESTS "${TARGET_LIBRARY}.Test")
+
+add_executable(${TARGET_LIBRARY_TESTS})
+
+target_sources(${TARGET_LIBRARY_TESTS} PRIVATE identity.t.cpp)
```

Comment:
Since we are already in a `test/` subdirectory, I don't think it is necessary to use `.t` in the filename.

Reply:
Moved tests near the production code.